server/service.go

```diff
@@ -0,0 +1,11 @@
+// +build !windows
+// Copyright 2012-2017 Apcera Inc. All rights reserved.
+
+package server
+
+// Run starts the NATS server. This wrapper function allows Windows to add a
+// hook for running NATS as a service.
+func Run(server *Server) error {
+	server.Start()
```

Comment:
Would it be better to handle this in `ConfigureLogger()`? The problem I see with turning on syslog here is it would get disabled on a config reload if the config file doesn't enable it. If we do the check in `ConfigureLogger()`, syslog will always be enabled even after a config reload. The downside of this is you wouldn't be able to disable it without specifying a log file, but I'm not sure if there's a use case for that.